### PR TITLE
Sign the EAR claims-set

### LINKED ISSUE #102: Proper signing of JSON Web Token during verification
Currently the JWT (JSON Web Token) returned by the verifier during verification only contains payload information and is missing header and signing information. The scope of this issue is to integrate JWT signing into the verifier so that the JWT encodes **signed** attestation results

### PR DESCRIPTION
**Pull request by thomas-fossati**
Fix #102

### CODE REVIEW

File: vts/earsigner/README.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+## Configuration
+
+- `ear-signer`: stanza containing the configuration details about the attestation
+  result signing process.  The supported directives are:
+  - `alg`: the [JWS algorithm](https://www.iana.org/assignments/jose/jose.xhtml#web-signature-encryption-algorithms)
+    used for signing, e.g.: `ES256`, `RS512`.
+  - `key`: file path of the private key to be used with `alg`.
```
Comment (yogeshbdeshpande) [inline vts/earsigner/README.md:7]: Is this a file path or a file (with its full location path) for the private key?

Could you please clarify it a bit?

**Review by yogeshbdeshpande — APPROVED** [review]
LGTM!

File: vts/earsigner/README.md (hunk 1 above)
Comment (thomas-fossati) [inline vts/earsigner/README.md:7]:
```suggestion
  - `key`: file containing the private key to be used with `alg`.
```

Comment (thomas-fossati) [reply vts/earsigner/README.md:7]: is this any clearer?

Comment (yogeshbdeshpande) [reply vts/earsigner/README.md:7]: thanks!

Comment (yogeshbdeshpande) [reply vts/earsigner/README.md:7]: yes!

**Review by SabreenKaur — APPROVED** [review]
Thank you!

**Pull request merged**